pinot-common/src/main/java/com/linkedin/pinot/common/utils/DataSchema.java

```diff
@@ -206,7 +206,7 @@ public int hashCode() {
   }
 
   public enum ColumnDataType {
-    INT, LONG, FLOAT, DOUBLE, STRING, OBJECT, INT_ARRAY, LONG_ARRAY, FLOAT_ARRAY, DOUBLE_ARRAY, STRING_ARRAY;
+    INT, LONG, FLOAT, DOUBLE, STRING, BYTES, OBJECT, INT_ARRAY, LONG_ARRAY, FLOAT_ARRAY, DOUBLE_ARRAY, STRING_ARRAY;
```

Comment:
Do we need it here? I don't think we allow serialize bytes between broker and server, and also we can reuse OBJECT for this.

Reply:
What I mean is that, BYTES should be one kind of OBJECT, instead of a separate ColumnDataType.

Reply:
That requires creation of ByteArray object when we could have just used byte[] primitive. This is just to avoid that additional wrapper object creation.